### DISCUSSION TITLE
just for command runner (instead of make)

### DISCUSSION THREAD

**Discussion by pjbull** [drivendataorg/cookiecutter-data-science#376] [Requested tools]
`just` is a tool for storing and running project specific commands.

Github: https://github.com/casey/just
Documentation: https://just.systems/man/en/

Pros:
 - Standalone binaries for all platforms
 - Simple syntax
 - Ability to create DAG
 - Seems pretty actively maintained
 - Supports .env files

Cons:
 - Not preinstalled in most environments
 - Not common enough that people will have experience already

--------------------

More discussion from #347 

 > make scares people, especailly of Windows, and I find https://github.com/casey/just command runner super helpful. it enables learning and you can try simple commands and extend them, extremely well-thought tool.

**Comment by jayqi** [comment]
It looks like someone made a PyPI package for distributing the Just binary. https://pypi.org/project/just-bin/

This means we could have it be a dependency for CCDS-generated projects. Though one would still need to install it first before being able to use it to manage project tasks.
